docs/docs/en/architecture/cache.md

```diff
@@ -2,9 +2,9 @@
 
 ## Purpose
 
-Due to the large database read operations during the master-server scheduling process. Such as read tables like `tenant`, `user`, `processDefinition`, etc. Operations stress read pressure to the DB, and slow down the entire core scheduling process.
+Due to the large database read operations during the master-server scheduling workflow. Such as read tables like `tenant`, `user`, `processDefinition`, etc. Operations stress read pressure to the DB, and slow down the entire core scheduling workflow.
```

Comment:
I think this content also should not change. Am I right @caishunfeng

Reply:
@caishunfeng Please confirm whether it needs to be modified